### ISSUE TITLE
deprecate weights = "insample.errors"

### ISSUE THREAD

**Issue by dashaub** [ellisp/forecastHybrid#42]
It probably never was a good idea to include this. Equal weights almost certainly lead to models with less variance, and cross validated error weights probably lead to lower model forecasting errors. It might be worth replacing this with the ability to set custom weights, e.g. 

```
hybridModel(wineind, models = “aen”, weights = c(0.2, 0.6, 0.2))
```

**Comment by dashaub** [comment]
@ellisp Thoughts on a soft depreciation of this? We can remove this from the choice of options provided in the documentation (i.e. `weights = c("equal", "cv.errors")`) but still preserve the in-sample behavior if called.

**Comment by ellisp** [comment]
Yes, I think we should do it that way, with a warning message too.  I was going to do this after the `thetaf` work gets merged, but happy for you to do it earlier.

**Comment by dashaub** [comment]
fixed here a50817a76e4ec75204f579e1ea16cecb72c2470a.
